auth-web/src/components/auth/common/ConfirmCancelButton.vue

```diff
@@ -62,27 +60,36 @@ export default class ConfirmCancelButton extends Vue {
   // for not to clear current org values [for account change , while clicking on cancel , current org has to stay]
   @Prop({ default: true }) clearCurrentOrg: boolean
 
-  private readonly resetAccountSetupProgress!: () => Promise<void>
+  @OrgModule.Action('setCurrentOrganizationFromUserAccountSettings') private setCurrentOrganizationFromUserAccountSettings!: () => Promise<any>
```

Comment:
Promise<void>? it returns nothing rite?

Reply:
good catch. I have changed it tp promise<void>